docs/blog/2019-03-25-behind-the-scenes-what-makes-gatsby-great/index.md

```diff
@@ -0,0 +1,496 @@
+  - This is often called "on the edge" and Gatsby can and should be deployed on the edge--[it reduces latency and improves page-load times](https://www.cloudflare.com/learning/cdn/glossary/edge-server/)
+- Creating a bucket/location to host static content (like S3, Google Cloud Storage, etc.)
+- Setting up a [Domain Name System (DNS)](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_domain_name) to make your content routable via a pretty URL, e.g. gatsbyjs.org
+- Routing the CDN layer in front of your static content
+
+... and, that's it! We're done as far as setup goes! We can use _whatever_ stack you prefer here. Netlify? [You bet.](https://www.gatsbyjs.org/docs/hosting-on-netlify) More a fan of AWS? [Of course.](https://www.gatsbyjs.org/docs/deploying-to-aws-amplify/) Heroku? [Yup.](https://www.gatsbyjs.org/docs/deploying-to-heroku/) You get the idea. Gatsby can be deployed anywhere easily and cheaply.
+
+We now have an infrastructure setup so that we _can_ release our web application. What's a release look like?
+
+```shell
+gatsby build
+```
+
+and then copying the result of `gatsby build` (the `public` folder) to your static content host of choice.
+
+That's it. What if your site goes viral and receives hundreds of thousands of requests? You get charged in **terrabytes** of data with most CDNs, and that cost is quite literally [pennies](https://aws.amazon.com/cloudfront/pricing/). Deploying a new version of your app? Fire off a deploy and your users will have the new version available the instant the build process completes.
+
+Let's contrast this approach with setting up and deploying a purely server-side rendered application.
+
+#### Deploying a server-side rendered application
+
+First: perform the **same** steps in setting up a static content host, excluding the step for configuring a bucket/location for hosting static content. Oh and we're not done yet, sweet, sweet summer child.
+
+![Sweet, Sweet Summer Child](https://media1.tenor.com/images/0ae8734d4f4184e8ac3d9f6032d0047f/tenor.gif?itemid=5580217)
+
+Next:
+
+- Set up and provision several [virtual machines (VM)](https://en.wikipedia.org/wiki/Virtual_machine) to run instance(s) of our server-side app
+  - Traditional examples are something like [EC2](https://aws.amazon.com/ec2/)
+- Configure the VM to be able to run Node.js code
+  - Consider using [Docker](https://www.docker.com/) to ensure you have a consistent Node.js environment
+- Set up auto-scaling to ensure we can accomodate and fall-over based upon heavy load or error states
+  - Consider using [Kubernetes](https://kubernetes.io/), [Rancher](https://rancher.com/), etc.
+- Set up a Continuous Integration (CI) environment so we can build and deploy code to production with minimal impact to end-users
+
+Does this sound like something that ✨joy? Oh--let's talk about the deploy process, too.
```

Comment:
```suggestion
Does this sound like something that sparks joy? ✨Oh--let's talk about the deploy process, too.
```
Moving the emoji to the end of the sentence so we don't rely on it for the sentence to read well.